x-pack/plugins/synthetics/server/legacy_uptime/lib/saved_objects/migrations/monitors/8.8.0.test.ts

```diff
@@ -117,11 +117,12 @@ describe('Monitor migrations v8.7.0 -> v8.8.0', () => {
           'ssl.supported_protocols': ['TLSv1.1', 'TLSv1.2', 'TLSv1.3'],
           'ssl.verification_mode': 'full',
           tags: [],
-          'throttling.config': '5d/3u/20l',
-          'throttling.download_speed': '5',
-          'throttling.is_enabled': true,
-          'throttling.latency': '20',
-          'throttling.upload_speed': '3',
+          throttling: {
```

Comment:
Please add specific tests in a new describe block to handle throttling. Please ensure it covers migrating default throttling values, custom throttling values, and no throttling from 8.7.0